### ISSUE TITLE
Missing Labels for QTable Pagination Buttons

### ISSUE THREAD

**Issue by gmeben** [quasarframework/quasar#17148]
### What happened?

The pagination buttons that appear at the bottom of QTables do not have label attributes. This violates [WCAG Success Criterion 4.1.2: Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html). 

![Screenshot of axe DevTools for Chrome on a CodePen with a basic QTable. Two 'Buttons must have discernible text' issues are detected. One of the two pagination buttons is highlighted for this issue.](https://github.com/quasarframework/quasar/assets/2668987/62815e84-0587-4360-8835-fd6a813963d8)

### What did you expect to happen?

Pagination buttons should have a detectable label attribute of some kind like `aria-label` or `title`.

### Reproduction URL

https://codepen.io/quadbaup/full/mdgoQWQ

### How to reproduce?

1. Visit the provided link to CodePen in Firefox 125
2. Authenticate on CodePen
3. Fork the linked CodePen
2. In the forked CodePen, change the view to "Debug" mode
4. Open Web Developer Tools
5. Open the Accessibility tab
6. Set the "Check for issues" dropdown to `All Issues`
7. Note the `button` Role issues:

> Text Labels and Names
> Error
> Interactive elements must be labeled. 
> [Learn more](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#interactive_elements_must_be_labeled)

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox
